ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -20,6 +20,14 @@ + (void)addContentsOfFolderToManifest:(NSString *)folderPath
     }
     
     for (NSString *fileName in folderFiles) {
+#if TARGET_IPHONE_SIMULATOR
```

Comment:
The IPA containing the app bundle, which a developer generates and then releases to the store, is still generated by the OSX machine right? From what I understand about this file, it is just used for saving icon display preferences for Finder. I'm not really sure how it got there in the first place even for the simulator, unless a user actually opened up the folder on disk using Finder. Either way, somehow OSX put it there when it generated the app bundle, so I presume its also entirely possible that it would generate it when making a release IPA for the device?